File: Plot_D.m
k = linspace(0,10,20);
[L,M] = monte_carlo(@sampleRandom, k , 1e5);
D = L./M;
N = 1./M;

figure;
plot(N,D);
title('Distortion');
xlabel('N(k)');
ylabel('D(k)');

File: MCsim-Mod1/MC-sim-julia-Mod1/Plot_D.jl
using PyPlot
include("monte_carlo.jl")
include("sampleThreshold.jl")
k = logspace(-5,1,200)
(L,M) = monte_carlo(sampleThreshold, k , 1e5)
D = L./M
N = 1./M
plot(N,D)
##show(N)
##show(D)
title("k=logspace(-5,1,200)")
xlabel("alpha")
ylabel("Distortion")
